docs/src/contributing/development-environment.md

```diff
@@ -1,280 +1,266 @@
 
-    To configure the registry for local Deis development:
-        export DEIS_REGISTRY=192.168.59.103:5000
+### Installing Deis Components
 
-It is important that you export the `$DEIS_REGISTRY` environment variable as instructed.
+With a functioning Kubernetes cluster, procede with Deis installation using [these instructions](../installing-deis/digitalocean.md) and then continue to follow the instructions below.
 
-If you are developing elsewhere, you must set up a registry yourself. Make sure it meets the following requirements:
+### Using a Development Registry
 
-1. You can push Docker images from your workstation
-2. Hosts in the cluster can pull images with the same URL
+To facilitate deploying Docker images containing your changes to your Kubernetes cluster, you will need to make use of a Docker registry.  This is a location to where you can push your custom-built images and from where your Kubernetes cluster can retrieve those same images.
 
-!!! note
-    If the development registry is insecure, then the Docker deamon on each of your nodes must be configured to allow communication with insecure registries in the applicable IP range.
+If your development cluster runs locally (in Vagrant, for instance), the most efficient and economical means of achieving this is to run a Docker registry locally _as_ a Docker container.
 
-## Initial Platform Build
+To facilitate this, most Deis components provide a make target to create such a registry:
 
-The full environment is prepared. You can now build Deis from source code and then run the platform.
+```
+$ make dev-registry
+```
 
-We'll do three steps together:
+In a Linux environment, to begin using the registry:
 
-- Build the source (`make build`)
-- Update our local cluster with a dev release (`make dev-release`)
-- Start the platform (`deisctl start platform`)
+```
+export DEIS_REGISTRY=<IP of the host machine>:5000
+```
 
-Conveniently, we can accomplish all three in one step:
+In non-Linux environments:
 
-    $ make deploy
+```
+export DEIS_REGISTRY=<IP of the docker-deis Docker Machine VM>:5000
+```
 
-Running `deisctl list` should display all of the services that your Deis cluster is currently running.
+If your development cluster runs on a cloud provider such as DigitalOcean, a local registry such as the one above will not be accessible to your Kubernetes nodes.  In such cases, a public registry such as [DockerHub][dh] or [quay.io][quay] will suffice.
 
-You can now use your Deis cluster in all of the usual ways.
+To use DockerHub for this purpose, for instance:
 
-At this point, you are running Deis from the code in your Git clone. But since rebuilding like this is time consuming, Deis has a simplified developer workflow more suited to daily development.
+```
+$ export DEIS_REGISTRY=docker.io
```

Comment:
Yup.  You're right.  This error was spotted in another PR, which erroneously led me to believe that might work.  It does not.  I will fix this.